docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,7 @@ metadata:
 spec:
   capacity:
     storage: 100Gi
+  volumeMode: Filesystem
```

Comment:
I would add a comment here saying that this field is only available if with the BlockVolume alpha feature

Reply:
Fixed.

Reply:
I think the official? term is "field", not "property"

Reply:
Fixed.